fe/fe-core/src/main/java/org/apache/doris/alter/SchemaChangeHandler.java

```diff
@@ -132,6 +132,26 @@ private void processAddColumn(AddColumnClause alterClause, OlapTable olapTable,
                           indexSchemaMap, newColNameSet);
     }
 
+    private void processAddColumn(AddColumnClause alterClause, Table externalTable, List<Column> newSchema) throws DdlException {
```

Comment:
```suggestion
    private void processAddCol{Alter}Operation(AddColumnClause alterClause, Table externalTable, List<Column> newSchema) throws DdlException {
```

Reply:
hi，yunfeng. I just want keep the function name same to the olaptable alter function name.

Reply:
That's ok